server/models/user.js

```diff
@@ -14,8 +14,8 @@ const userSchema = new Schema({
   groups: [{ type: Schema.Types.ObjectId, ref: 'Group' }],
   usersIFollow: [{ type: Schema.Types.ObjectId, ref: 'User' }],
   usersFollowingMe: [{ type: Schema.Types.ObjectId, ref: 'User' }],
-  exploreNumber: Number,
-  numExplorations: Number,
+  exploreNumber: { type: Number, default: 0 },
+  numExplorations: { type: Number, default: 20 },
```

Comment:
ah.

Neither - the idea is that if they use the explore plugin/algorithm and populate with facebook data, then we say "numExplorations" is 20 so after they've annotated 20 articles, their exploreNumber will be 50% from facebook data and 50% from notist data, so therefore the facebook data will become less and less important. BUT, Now that I think about it, it doesn't make sense for default to be 20... so I'll change.